package.json

```diff
@@ -101,29 +101,29 @@
     "clean-webpack-plugin": "^4.0.0",
     "copy-webpack-plugin": "^11.0.0",
     "copyfiles": "^2.4.1",
-    "core-js": "^3.32.1",
+    "core-js": "^3.33.0",
     "cross-env": "^7.0.3",
     "del": "^6.1.1",
     "elliptic": "^6.5.4",
-    "eslint": "^8.48.0",
+    "eslint": "^8.51.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-babel": "^5.3.1",
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-prettier": "^5.0.1",
```

Comment:
I know we were at `v5.0.0` previously but can we actually bump this down to `v4.2.1`? I realized that eslint-plugin-prettier v5 is expecting that we have prettier v3 as a peerDependency

Reply:
have we tried upgrading to prettier v3?

Reply:
Yeah I tried upgrading to v3 last month but it wasn't working